syntax = "proto3";
package todo;

message TodoItem {
	string title = 1;
	string short_description = 2;
	string description = 3;
	string created_by = 4;
}

message AddTodoResponse {
	int32 addedTodo = 1;
	string message = 2;
	Error error = 3;
}

message Error {
	string code = 1;
	string message = 2;
}

service TodoService {
	rpc AddTodo(TodoItem) returns (AddTodoResponse) {}
}

